src/components/v5/shared/MeatBallMenu/types.ts

```diff
@@ -0,0 +1,12 @@
+export interface MeatBallItem {
+  key: string;
+  label: React.ReactNode;
+  iconName: string;
```

Comment:
there are only 3 types of menu items available in the entire application: add, duplicate, delete. The icons are the same for all tables in all action types. Shouldn't we have this icons inside this component rather pass it every time when this component will be used?

Reply:
@krawczykkatarzyna we want this to be reusable, so this approach is ok. Later we can create a component with specific items and pass other props there, for example as a partial for a table with actions in such menu